test/functional/shared/resources/testdata/recipes/test-bicep-recipes/README.md

```diff
@@ -2,7 +2,7 @@
 
 The recipes in this folder are published as part of the PR process to:
 
-> `radiusdev.azurecr.io/test/functional/shared/recipes/<filename>:pr-<pr-number>`
+> `ghcr.io/radius-project/dev/test/functional/shared/recipes/<filename>:pr-<pr-number>`
 
```

Comment:
I found one more instance of azurecr.io:-

![Screenshot 2023-10-17 at 10 46 04 AM](https://github.com/radius-project/radius/assets/28875764/b06429dc-f592-400a-aa20-f67216bba29e)

Reply:
good catch, thanks